app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.controller.spec.ts

```diff
@@ -1,102 +0,0 @@
-import { KubernetesV2DeployManifestConfigCtrl as Controller } from './deployManifestConfig.controller';
```

Comment:
The `YamlEditor` is now responsible for translating JS objects to YAML, rather than the parent controllers. My understanding of this test is that it tested that behavior for the deploy manifest controller.